src/site/content/en/blog/codelab-payment-form-best-practices/index.md

```diff
@@ -0,0 +1,340 @@
+---
+layout: codelab
+title: Payment form best practices codelab
+authors:
+  - samdutton
+scheduled: true
+date: 2020-12-02
+updated: 2020-12-02
+description: Learn best practices for payment forms.
+tags:
+  - ecommerce
+  - forms
+  - identity
+  - layout
+  - mobile
+  - security
+  - ux
+glitch: payment-form-codelab-0
+glitch_path: index.html
+related_post: payment-and-address-form-best-practices
+---
+
+This codelab shows you how to build a payment form that is secure, accessible and easy to use.
+
+{% Aside 'caution' %}
+This article is about frontend best practices for payment forms. It does not explain how to 
+implement transactions on your site. To find out more about adding payment functionality to your 
+website, see [Web Payments](/payments).
+{% endAside %}
+
+## Step 1: Use HTML as intended
+
+Use elements built for the job: 
+* `<form>`
+* `<section>`
+* `<label>`
+* `<input>`, `<select>`, `<textarea>`
+* `<button>`
+
+As you'll see, these elements enable built-in browser functionality, improve accessibility, and 
+add meaning to your markup.
+
+{% Instruction 'remix' %}
+
+* Add the following code inside the `<body>` element:
```

Comment:
Let's put the starter code in Glitch like in the sing-up form codelab.

Reply:
Good idea.

Updated the text here (and set the starter code to include the form HTML).